qiskit_algorithms/optimizers/nlopts/sbplx.py

```diff
@@ -0,0 +1,37 @@
+# This code is part of a Qiskit project.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Sbplx (Subplex) optimizer."""
+
+from .nloptimizer import NLoptOptimizer, NLoptOptimizerType
+
+
+class SBPLX(NLoptOptimizer):
+    """
+    Subplex optimizer.
+
+    "Subplex (a variant of Nelder-Mead that uses Nelder-Mead on a sequence of subspaces)
+    is claimed to be much more efficient and robust than the original Nelder-Mead,
+    while retaining the latter's facility with discontinuous objectives,
+    and in my experience these claims seem to be true in many cases.
+    (However, I'm not aware of any proof that Subplex is globally convergent,
+    and perhaps it may fail for some objectives like Nelder-Mead; YMMV.)"
+    Description by Steven G. Johnson, author of NLopt library.
```

Comment:
Its failing the spell check here ` Description by Steven G. Johnson, author of NLopt library.` on `steven`  and `johnson` which should be added similarly to the other couple of words you just added to the custom dict. I am not sure why it did not show these the first time - maybe spelling stops after a certain number of checks, I cannot recall.

>qiskit_algorithms/optimizers/nlopts/sbplx.py:docstring of qiskit_algorithms.optimizers.nlopts.sbplx.SBPLX:9: : Spell check: steven: Description by Steven G. Johnson, author of NLopt library..
qiskit_algorithms/optimizers/nlopts/sbplx.py:docstring of qiskit_algorithms.optimizers.nlopts.sbplx.SBPLX:9: : Spell check: johnson: Description by Steven G. Johnson, author of NLopt library..

Reply:
No idea either, why it didn't show up in the first run. Either way, I fixed it.